docs/network_firewall.rst

```diff
@@ -99,6 +99,38 @@ chosen:
 -  Monitor Gateway: ``10.20.3.1``
 -  Monitor Server (OPT2) : ``10.20.3.2``
 
+3 NIC Example (SG-3100)
+'''''''''''''''''''''''
+
+As described earlier, the SG-3100 has an internal switch on the LAN interface
+which means we can place the *Application Server* and *Admin Workstation* on
+the same subnet and gateway. This example assumes you place the
+*Admin Workstation* on LAN1 and the *Application Server* on LAN2.
```

Comment:
I don't think it matters which LAN ports the admin workstation and application server are connected to (as long as it's one of the four LAN ports) since it's a simple switch.

Reply:
It does not. I was slightly confused about which port to put the App and Admin servers so I figured I would be more verbose just to help future admins.

Reply:
I just removed the line `This example assumes you place the *Admin Workstation* on LAN1 and the *Application Server* on LAN2.` As you said, it shouldn't matter but i think having too much language could be confusing.